core/modules/replica/HttpProcessor.h

```diff
@@ -248,8 +248,6 @@ class HttpProcessor : public std::enable_shared_from_this<HttpProcessor> {
     // the reference counters to the pointed objects.
 
     HealthMonitorTask::Ptr const& _healthMonitorTask;
-    ReplicationTask::Ptr   const& _replicationTask;
-    DeleteWorkerTask::Ptr  const& _deleteWorkerTask;
```

Comment:
This is good for now. Though, I will need to put both back later when I will actually need these references. An idea behind having these references here was to allow REST API to suspend/resume those tasks if needed.